README.md

```diff
@@ -77,6 +77,11 @@ npm install chrome-launcher
   // Default: 50
   maxConnectionRetries: number;
 
+  // (optional) Interval in ms, which defines whether and how long an inspectable target should be awaited.
+  // `0` means that list of inspectable targets will not be requested and awaited.
+  // Default: 0
+  waitForInspectableTarget: number;
```

Comment:
Wondering if this ought to be the default behavior. about to do a major version anyhow, so we can do a breaking change. Will play with this more later.